tasks/src/conversational/data.ts

```diff
@@ -48,7 +48,12 @@ const taskData: TaskDataCustom = {
 			id:          "microsoft/DialoGPT-large",
 		},
 	],
-	spaces:       [],
+	spaces:       [
+		{
+			description: "DialoGPT is a large-scale pretrained dialogue response generation model for multiturn conversations.",
+			id:          "microsoft/DialoGPT-large",
```

Comment:
This is a model id, not a spaces id. Maybe use https://huggingface.co/spaces/mandar100/chatbot_dialogpt here